sql/_23_apricot_qa/_02_performance/_02_function_based_index/answers/function_based_index_coverage_003.answer

```diff
@@ -1,26 +1,23 @@
 ===================================================
```

Comment:
The result of this test case is wrong.
Please check the test result again.

Reply:
Reflected the review. Please check again.